docker/Readme.md

```diff
@@ -37,7 +37,7 @@ We have also developed a tool on top of docker that you could be interested in:
 
 
 ### Other Dockerfiles related with Robocomp
-This other Dockerfiles, dockercompose files and related images in docker hub could be of your interest if you are working with these projects.
+These other Dockerfiles, docker-compose files, and related images in the docker hub could be of your interest if you are working with these projects.
```

Comment:
@orensbruli  I have one doubt related to the suggested changes that , I didn't particularly understood what change you want me to do in  "if" word . According to me it's grammatically correct.